hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAccountWipeHandler.java

```diff
@@ -54,8 +84,123 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         }
     }
 
+    @Override
+    public void pureChecks(@NonNull final TransactionBody txn) throws PreCheckException {
+        final var op = txn.tokenWipeOrThrow();
+
+        // All the pure checks for burning a token must also be checked for wiping a token
+        burnPureChecks(op.amount(), op.serialNumbers(), op.hasToken(), INVALID_WIPING_AMOUNT);
```

Comment:
Method should be named differently than `burnPureChecks`.

Reply:
I couldn't think of anything better. `burnAndWipePureChecks` Sounded worse. Any ideas?

Reply:
I couldn't think of anything better. `burnAndWipePureChecks` Sounded worse. Any ideas?

Reply:
I couldn't think of anything better. `burnAndWipePureChecks` Sounded worse. Any ideas?

Reply:
May be `commonChecksForBurnWipeOps` ?

Reply:
Renamed